crates/api/src/space.rs

```diff
@@ -91,6 +94,19 @@ pub trait Space: 'static + Send + Sync + std::fmt::Debug {
         from_agent: AgentId,
         data: bytes::Bytes,
     ) -> BoxFut<'_, K2Result<()>>;
+
+    /// Inform the space that a set of ops have been stored.
+    ///
+    /// This should be called once the ops have been:
+    /// - Checked to be correctly structured according to the Kitsune2 host implementation.
+    /// - Persisted to the op store.
+    ///
+    /// Until this is called, gossip will not include these ops in its DHT model, and they will
+    /// therefore not be synced with other peers. They may be synced with other peers through the
+    /// "new ops" mechanism, depending on the op store implementation, but that only makes them
+    /// available briefly.
+    fn inform_ops_stored(&self, ops: Vec<StoredOp>)
```

Comment:
This will need to be available for the host to call anyway, I just hadn't added it yet. So this solves the problem of the operation not being available on the K2Gossip when it's behind a DynGossip